src/app/static/src/app/store/filteredData/getters.ts

```diff
@@ -98,13 +94,9 @@ export const getters = {
 
         const indicators = Object.keys(indicatorsMeta);
 
-        for(const d of data) {
-            const row = d as any;
+        for (const row of data) {
 
-            if (!includeRowForSelectedChoroplethFilters(row,
-                state.selectedDataType,
-                state.selectedChoroplethFilters,
-                flattenedRegions)) {
+            if (getters.excludeRow(row)) {
```

Comment:
Sorry this was a bit of scope creep, but I thought this made the code neater - making the filter function a getter so we don't have to pass all the state related arguments, and inverting the logic.